src/overlays/actors/ovl_Bg_Ingate/z_bg_ingate.h

```diff
@@ -4,14 +4,14 @@
 #include "ultra64.h"
 #include "global.h"
 
-struct BgIngate;
+struct BgIngoGate;
```

Comment:
I don't know about renaming this actor right now. So far we've been using the overlay name everywhere so this is a pretty unusual change at least